Sprint-2/5-stretch-extend/format-time.js

```diff
@@ -23,3 +23,31 @@ console.assert(
   currentOutput2 === targetOutput2,
   `current output: ${currentOutput2}, target output: ${targetOutput2}`
 );
+
+const currentOutput3 = formatAs12HourClock("00:00");
+const targetOutput3 = "12:00 am";
+console.assert(
+  currentOutput3 === targetOutput3,
+  `Test failed :00:00 current output: ${currentOutput3}, target output: ${targetOutput3}`
```

Comment:
This assertion is failing for me. What do we have to change in `formatAs12HourClock` to fix it?